watertap/costing/unit_models/gac.py

```diff
@@ -101,6 +102,11 @@ def cost_gac_gravity(blk):
 def _build_gac_cost_param_block(blk, contactor_type):
 
     adsorbent_unit_cost_coeff_data = {0: 4.58342, 1: -1.25311e-5}
+    contactor_cost_coeff_data, other_cost_param_data, energy_consumption_coeff_data = (
+        {},
+        {},
+        {},
+    )
```

Comment:
I think this is another one of those times we should raise an exception in an `else`. Otherwise we will get an error elsewhere and it'll be difficult to trace back here.

Reply:
changed this to an else